MagazineLayout/LayoutCore/Types/TargetContentOffsetAnchor.swift

```diff
@@ -22,25 +22,28 @@ import UIKit
 enum TargetContentOffsetAnchor: Equatable {
   case top
   case bottom
-  case topItem(id: String, distanceFromTop: CGFloat)
-  case bottomItem(id: String, distanceFromBottom: CGFloat)
+  case topItem(id: String, itemEdge: ItemEdge, distanceFromTop: CGFloat)
+  case bottomItem(id: String, itemEdge: ItemEdge, distanceFromBottom: CGFloat)
```

Comment:
this change allows us to preserve the scroll position more accurately when cells change size on rotation (their height might grow / shrink depending on the width after rotation)